src/modules/RA/Viewpoint/components/ViewpointFields.js

```diff
@@ -56,16 +59,24 @@ const ViewpointFields = ({ edit, classes, ...props }) => {
   };
 
   React.useEffect(() => {
-    if (edit) {
+    if (edit || create) {
```

Comment:
As if edit OR create should be always truthy, condition id now useless. This could be simplfy to:

```js
React.useEffect(() => {
  getRemoteData();
}, []);
```

Reply:
Done